src/main/java/utils/DBConnector.java

```diff
@@ -0,0 +1,27 @@
+package utils;
+
+import org.apache.log4j.Logger;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.TimeZone;
+
+public class DBConnector {
+
+    private static final Logger logger = Logger.getLogger(DBConnector.class);
+    private static final String dbUrl = "jdbc:mysql://localhost:3306/online_shop?serverTimezone=" +
```

Comment:
Hoe do you think, are this field: dbUrl, login, password constants?

Reply:
fixed. made them non-final.

Reply:
I think them can be constants, but then we need write name like this: DB_URL